lasagne/tests/layers/test_conv.py

```diff
@@ -181,20 +185,40 @@ def _convert(input, kernel, output, kwargs):
                                                        'stride': stride,
                                                        'flip_filters': True})
 
+    # extend (w/ and w/out symbolic output shape)
+    for symbolic in [False, True]:
+        input_shape = (4, 3, 7, 9)
+        input = np.random.random(input_shape)
+        kernel = np.random.random((16, 3, 2, 3))
+        stride = np.random.randint(2, 3, 2)
+        extend = [np.random.randint(0, s-1) for s in stride]
```

Comment:
Again, I'm not sure about using randomness here. Maybe fix the stride to (2,3) and loop over two choices for extend, like (0,1) and (1,2).